src/DatePicker/DatePickerNavbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Box from '../Box';
+import Button from '../Button';
+import Flex from '../Flex';
+import Icon from '../Icon';
+
+const DatePickerNavbar = ({ onPreviousClick, onNextClick, className }) => (
+  <Flex className={className} alignItems="center">
+    <Button
+      iconStart={<Icon color="icon.default" name="chevron" rotation="90" />}
+      onClick={() => onPreviousClick()}
+      size="small"
+      variant="minimal"
+    />
+    <Box mr="small" ml="small">
+      <Icon color="icon.default" fontSize="size5" name="symbol-circle" />
+    </Box>
+    <Button
+      iconStart={<Icon color="icon.default" name="chevron" rotation="270" />}
+      onClick={() => onNextClick()}
```

Comment:
Similar question here, could this just be `onClick={onNextClick}`?

Reply:
Same scope change issue.